### PR TITLE
Update marketplace for cluster mode

### PR DESCRIPTION
**Pull request by gregaubert**

### CODE REVIEW

**Comment by gregaubert** [comment]
@stas-vilchik you can review only the last two commits

File: server/sonar-web/src/main/js/apps/marketplace/components/EditionsStatusNotif.tsx [hunk 1]
```diff
@@ -49,7 +50,8 @@ export default class EditionsStatusNotif extends React.PureComponent<Props, Stat
   };
 
   renderStatusAlert() {
```
Comment (SonarTech) [inline server/sonar-web/src/main/js/apps/marketplace/components/EditionsStatusNotif.tsx]: CRITICAL The function renderStatusAlert has a cyclomatic complexity of 11 which is higher than the threshold of 10 [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1541)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [PluginsList.tsx#L53](https://github.com/SonarSource/sonarqube/blob/a31a7e7402936efca8417571bc97ec295b454ec2/server/sonar-web/src/main/js/apps/marketplace/PluginsList.tsx#L53): Refactor this function to use "return" consistently [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3801)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 3 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [App.tsx#L150](https://github.com/SonarSource/sonarqube/blob/096c123cadf014d7d079f889acad5e8b0a862183/server/sonar-web/src/main/js/apps/marketplace/App.tsx#L150): Update this function so that its implementation is not identical to the one on line 134. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS4144)
1. ![MAJOR][MAJOR] [PluginsList.tsx#L53](https://github.com/SonarSource/sonarqube/blob/096c123cadf014d7d079f889acad5e8b0a862183/server/sonar-web/src/main/js/apps/marketplace/PluginsList.tsx#L53): Refactor this function to use "return" consistently [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3801)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: server/sonar-web/src/main/js/apps/marketplace/components/PluginActions.tsx [hunk 2]
```diff
@@ -69,47 +69,53 @@ export default class PluginActions extends React.PureComponent<Props, State> {
   handleUninstall = () => this.doPluginAction(uninstallPlugin);
   handleTermsCheck = (checked: boolean) => this.setState({ acceptTerms: checked });
 
+  renderBundled() {
```
Comment (SonarTech) [inline server/sonar-web/src/main/js/apps/marketplace/components/PluginActions.tsx]: CRITICAL The function render has a cyclomatic complexity of 15 which is higher than the threshold of 10 [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1541)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [App.tsx#L150](https://github.com/SonarSource/sonarqube/blob/f08cb6ecbb3465d01e3532262071af4a63e51abb/server/sonar-web/src/main/js/apps/marketplace/App.tsx#L150): Update this function so that its implementation is not identical to the one on line 134. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS4144)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [App.tsx#L150](https://github.com/SonarSource/sonarqube/blob/f08cb6ecbb3465d01e3532262071af4a63e51abb/server/sonar-web/src/main/js/apps/marketplace/App.tsx#L150): Update this function so that its implementation is not identical to the one on line 134. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS4144)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**